app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
add .idea, .gradle, build, build directories to this and files like local.properties...